GNUmakefile

```diff
@@ -13,6 +13,9 @@ endif
 ifndef STARKNET_SHA
 override STARKNET_SHA = "9a780650af4708e4bd9b75495feff2c5b4054e46"
 endif
+ifndef APTOS_RELAYER_GIT_REF
+override APTOS_RELAYER_GIT_REF = "213b9555e7ece0e7988d2aea844ad8d1d736201a"
```

Comment:
We should update this with a release tag when available.

Reply:
We currently run `tag: "2.21.0-beta16-aptos"` on prod-testnet: https://github.com/smartcontractkit/infra-k8s/blob/main/projects/chainlink/files/chainlink-clusters/cl-keystone-cap-one/config.yaml#L221

This image tag is published from here: https://github.com/smartcontractkit/chainlink-internal-integrations/tree/2.21.0-beta16-aptos

This should be the Aptos plugin we bundle in initially.

Then finally we will need to prepare and accept a new plugin release from https://github.com/smartcontractkit/chainlink-aptos (new code location), and at this point we can switch.

Reply:
Sounds good. Based on that then, I'll set the git ref to `2.21.0-beta16-aptos` here.

Reply:
359730fbc1